string-hash/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for string-hash 1.1.1
+// Project: https://github.com/darkskyapp/string-hash
+// Definitions by: Ethan Rubio <https://github.com/ethanrubio/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace StringHash {
```

Comment:
That error just means you should put `declare` in front of `function`. (Same goes if you see this error in front of `namespace` or `class`).